vmdb/app/controllers/ops_controller/settings/schedules.rb

```diff
@@ -638,7 +638,7 @@ def build_db_options_for_select
   def schedule_set_basic_record_vars(schedule)
     schedule.name = params[:name]
     schedule.description = params[:description]
-    schedule.enabled = params[:enabled]
+    schedule.enabled = params[:enabled] || "off"
```

Comment:
@romanblanco verified fix in UI, just curious why are we using "off" here instead of "false", that's what we usually use.